webapi/src/main/scala/org/knora/webapi/OntologyConstants.scala

```diff
@@ -749,6 +749,9 @@ object OntologyConstants {
 
         val ResourceIcon: IRI = KnoraApiV2PrefixExpansion + "resourceIcon"
 
+        // TODO: Add this to the ontology in the complex schema
```

Comment:
Could you do this TODO?

Reply:
I have already added it to `KnoraApiV2WithValueObjects.scala`, just forgotten to remove the TODO :-).

Reply:
removed TODO in https://github.com/dhlab-basel/Knora/pull/811/commits/2d8e33d09b0c77f5916ef1fb3e6aabe1e9fb61ea